h/static/scripts/annotator/range.coffee

```diff
@@ -493,8 +450,8 @@ class Range.SerializedRange
   # Public: Returns the range as an Object literal.
   toObject: ->
     {
-      startContainer: @startContainer
+      start: @start
       startOffset: @startOffset
-      endContainer: @endContainer
+      end: @end
```

Comment:
Is this not going to break existing annotations? I can't see a transform function.